### PR TITLE
feat: `requestBody` types

### LINKED ISSUE #338: requestBody parameters are missing
For example, GitHub's example schema (`tests/v3/specs/github.yaml`) has lots of `requestBody` parameters. But the generated TypeScript types (`tests/v3/expected/github.ts`) do not have any of it

As suggested in my comment here: https://github.com/drwpow/openapi-typescript/issues/236#issuecomment-701054647, I'd transform something like this

```yml
      requestBody:
        content:
          application/json:
            schema:
              type: object
              properties:
                name:
                  type: string
                description:
                  type: string
                homepage:
                  type: string
                private:
                  type: boolean
                  default: false
              required:
                - name
```

into

```ts
      requestBody: {
        "application/json": {
          name: string;
          description?: string;
          homepage?: string;
          private?: boolean;
        };
      };
```

### PR DESCRIPTION
**Pull request by gr2m**
fixes #338

### CODE REVIEW

File: tests/bin/expected/petstore.ts [hunk 1]
```diff
@@ -194,6 +226,9 @@ export interface paths {
   }
   '/store/order': {
     post: {
+      requestBody: {
+        '*/*': components['schemas']['Order']
```
Comment (drwpow) [inline tests/bin/expected/petstore.ts:230]: Ah interesting. Didn’t even know you could do this

File: tests/v3/index.test.ts [hunk 2]
```diff
@@ -630,4 +630,66 @@ describe("OpenAPI3 features", () => {
...
+              "201": {
+                'application/json': { id: string; title: string}
+              };
+            };
+          };
+        };
+      }
+
+      export interface components {}
+    `)
```
Comment (drwpow) [inline tests/v3/index.test.ts:692]: 👍🏻

**Review by drwpow — APPROVED** [review]
🔥 🔥 🔥 You’re on fire! 😄

File: tests/bin/expected/petstore.ts (hunk 1 above)
Comment (gr2m) [reply tests/bin/expected/petstore.ts:230]: I'm not exactly sure if this is part of the specification, but GitHub is using it for its ["Upload a release asset" endpoint](https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#upload-a-release-asset), which allows you to upload any kind of file, of any type

**Pull request closed without merging**